src/app/year-calendar/year-calendar-config.ts

```diff
@@ -0,0 +1,28 @@
+import {locale, Moment} from 'moment';
+import {ICalendar, ICalendarInternal} from '../common/models/calendar.model';
+import {ECalendarValue} from '../common/types/calendar-value-enum';
+
+export interface IConfig {
+  isYearDisabledCallback?: (date: Moment) => boolean;
+  allowMultiSelect?: boolean;
+  yearFormat?: string;
+  yearFormatter?: (month: Moment) => string;
+  format?: string;
+  isNavHeaderBtnClickable?: boolean;
+  yearBtnFormat?: string;
+  yearBtnFormatter?: (day: Moment) => string;
+  yearBtnCssClassCallback?: (day: Moment) => string;
+  multipleYearsNavigateBy?: number;
+  showMultipleYearsNavigation?: boolean;
+  returnedValueType?: ECalendarValue;
+  showGoToCurrent?: boolean;
+}
+
+export interface IYearCalendarConfig extends IConfig,
+                                              ICalendar {
+
```

Comment:
redundant

Reply:
OK

Reply:
... however, this is identical to `month-calendar-config.ts`. I don't understand your comment, can I simply remove the `IYearCalendarConfig` interface?